app/ui/browser-modern/setup-ipc.js

```diff
@@ -47,19 +48,25 @@ export default function({ store, userAgentClient }) {
   });
 
   ipcRenderer.on('go-back', () => {
-    if (!UISelectors.getOverviewVisible(store.getState())) {
+    const pageId = PagesSelectors.getSelectedPageId(store.getState());
+    const pageState = PagesSelectors.getPageState(store.getState(), pageId);
```

Comment:
These two lines are a little clumsy compared to `PagesSelectors.getSelectedPage(store.getState()).state`.  Does hiding the `state` property inside getPageState provide much value, given that we will immediately need to dig into the state object's internals?